.github/workflows/run_integration_tests.yml

```diff
@@ -93,10 +95,10 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Setup .NET 6.0
-        uses: actions/setup-dotnet@v1
+      - name: Add .NET 7 SDK
```

Comment:
We don't have to do this now, but the 2022 runner already includes .NET 7: https://github.com/actions/runner-images/blob/main/images/win/Windows2022-Readme.md#net-core-tools

Reply:
@chynesNR I went ahead and made that change, both in this file and in `all_solutions.yml` - there were several references that I removed, so maybe the build will be a few seconds faster now... ;-)